### PR TITLE
Adjust documentation after removing jaeger

### PR DESCRIPTION
**Pull request by rakesh-garimella**
**Description**

Changes proposed in this pull request:

- Remove references to Jaeger as it has been deleted from kyma
- Adapt serverless documentation
- Remove/adapt obsolete jaeger references

**Related issue(s)**

https://github.com/kyma-project/kyma/issues/16584

### CODE REVIEW

File: docs/01-overview/main-areas/service-mesh/smsh-03-istio-sidecars-in-kyma.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ Another security benefit of having a sidecar proxy is that you can perform [requ
 
 ## Observability
 
-Furthermore, Istio proxies improve tracing: Istio performs global tracing and forwards the data to [Kyma's tracing component](../../../01-overview/main-areas/observability/obsv-03-tracing-in-kyma.md) using the [Zipkin protocol](https://zipkin.io). Learn more about the process in [Tracing Architecture](../../../05-technical-reference/00-architecture/obsv-03-architecture-tracing.md).
+Furthermore, Istio proxies improve tracing: Istio performs global tracing and forwards the data to a tracing backend. One can bring own tracing backend or one can deploy [Jaeger](https://github.com/kyma-project/examples/tree/main/jaeger) inside the cluster using the [Zipkin protocol](https://zipkin.io). Learn more about  in [Tracing Architecture](../../main-areas/telemetry/telemetry-03-traces.md).
```
Comment (chrkl) [inline docs/01-overview/main-areas/service-mesh/smsh-03-istio-sidecars-in-kyma.md:21]: Let's replace Zipkin by OTLP here. That's what all the defaults for Istio tracing expect now and also the preferred way with Jaeger.

File: docs/03-tutorials/00-serverless/svls-12-customize-function-traces.md [hunk 2]
```diff
@@ -2,16 +2,17 @@
...
+Kyma Functions are instrumented to handle trace headers. This means that every time you call your Function, the executed logic is traceable using a dedicated span visible in the trace backend (i.e. start time and duration).
 Additionally, you can extend the default trace context and create your own custom spans whenever you feel it is helpful (i.e. when calling a remote service in your distributed application) or add additional information to the tracing context by introducing events and tags. This tutorial shows you how to do it using tracer client that is available as part of the [event](../../05-technical-reference/svls-08-function-specification.md#event-object) object.
 
 ## Prerequisites
 
 Before you start, make sure you have these tools installed:
 
 - [Kyma installed](../../04-operation-guides/operations/02-install-kyma.md) on a cluster
+- Trace backend configured to collect traces from the cluster. Kyma allows bringing an own trace backend or one can deploy [jaeger](https://github.com/kyma-project/examples/tree/main/jaeger).
```
Comment (chrkl) [inline docs/03-tutorials/00-serverless/svls-12-customize-function-traces.md:15]:
```suggestion
- Trace backend configured to collect traces from the cluster. Kyma allows bringing an own trace backend or one can deploy [Jaeger](https://github.com/kyma-project/examples/tree/main/jaeger).
```

File: docs/01-overview/main-areas/observability/obsv-10-useful-links.md [hunk 3]
```diff
@@ -17,10 +17,10 @@ If you're interested in learning more about the Observability area, check out th
...
-  - [Jaeger shows fewer traces than expected](../../../04-operation-guides/troubleshooting/observability/obsv-02-troubleshoot-jaeger-shows-few-traces.md)
+  - [Tracing backend shows fewer traces than expected](../../../04-operation-guides/troubleshooting/observability/obsv-02-troubleshoot-trace-backend-shows-few-traces.md)
   - [Loki shows fewer logs than expected](../../../04-operation-guides/troubleshooting/observability/obsv-03-troubleshoot-loki-logging.md)
 
-- Understand the architecture of Kyma's [monitoring](../../../05-technical-reference/00-architecture/obsv-01-architecture-monitoring.md), [logging](../../../05-technical-reference/00-architecture/obsv-02-architecture-logging.md), and [tracing](../../../05-technical-reference/00-architecture/obsv-03-architecture-tracing.md) components.
+- Understand the architecture of Kyma's [monitoring](../../../05-technical-reference/00-architecture/obsv-01-architecture-monitoring.md), [logging](../../../05-technical-reference/00-architecture/obsv-02-architecture-logging.md), and [tracing](../../main-areas/telemetry/telemetry-03-traces.md) components.
```
Comment (NHingerl) [inline docs/01-overview/main-areas/observability/obsv-10-useful-links.md:23]: Why was the word "architecture" removed from the new file name? Isn't the "new" document about architecture anymore?

File: docs/01-overview/main-areas/service-mesh/smsh-03-istio-sidecars-in-kyma.md [hunk 4]
```diff
@@ -18,7 +18,7 @@ Another security benefit of having a sidecar proxy is that you can perform [requ
 
 ## Observability
 
-Furthermore, Istio proxies improve tracing: Istio performs global tracing and forwards the data to [Kyma's tracing component](../../../01-overview/main-areas/observability/obsv-03-tracing-in-kyma.md) using the [Zipkin protocol](https://zipkin.io). Learn more about the process in [Tracing Architecture](../../../05-technical-reference/00-architecture/obsv-03-architecture-tracing.md).
+Furthermore, Istio proxies improve tracing: Istio performs global tracing and forwards the data to a tracing backend. One can bring own tracing backend or one can deploy [Jaeger](https://github.com/kyma-project/examples/tree/main/jaeger) inside the cluster using the [Otlp protocol](https://opentelemetry.io/docs/reference/specification/protocol/). Learn more about  in [Tracing Architecture](../../main-areas/telemetry/telemetry-03-traces.md).
```
Comment (NHingerl) [inline docs/01-overview/main-areas/service-mesh/smsh-03-istio-sidecars-in-kyma.md:21]:
```suggestion
Furthermore, Istio proxies improve tracing: Istio performs global tracing and forwards the data to a tracing backend. You can bring own trace backend or deploy [Jaeger](https://github.com/kyma-project/examples/tree/main/jaeger) inside the cluster using the [OTLP protocol](https://opentelemetry.io/docs/reference/specification/protocol/). Learn more in [Tracing Architecture](../../main-areas/telemetry/telemetry-03-traces.md).
```

File: docs/03-tutorials/00-serverless/svls-12-customize-function-traces.md [hunk 5]
```diff
@@ -2,16 +2,17 @@
...
+Kyma Functions are instrumented to handle trace headers. This means that every time you call your Function, the executed logic is traceable using a dedicated span visible in the trace backend (i.e. start time and duration).
 Additionally, you can extend the default trace context and create your own custom spans whenever you feel it is helpful (i.e. when calling a remote service in your distributed application) or add additional information to the tracing context by introducing events and tags. This tutorial shows you how to do it using tracer client that is available as part of the [event](../../05-technical-reference/svls-08-function-specification.md#event-object) object.
 
 ## Prerequisites
 
 Before you start, make sure you have these tools installed:
 
 - [Kyma installed](../../04-operation-guides/operations/02-install-kyma.md) on a cluster
+- Trace backend configured to collect traces from the cluster. Kyma allows bringing an own trace backend or one can deploy [Jaeger](https://github.com/kyma-project/examples/tree/main/jaeger).
```
Comment (NHingerl) [inline docs/03-tutorials/00-serverless/svls-12-customize-function-traces.md:15]:
```suggestion
- Trace backend configured to collect traces from the cluster. You can bring your own trace backend or deploy [Jaeger](https://github.com/kyma-project/examples/tree/main/jaeger).
```

File: docs/04-operation-guides/security/sec-06-access-expose-grafana.md [hunk 6]
```diff
@@ -74,66 +53,24 @@ The following example shows how to use an OpenID Connect (OIDC) compliant identi
...
-  <details>
-  <summary>
-  Jaeger
-  </summary>
+>**NOTE:** By default, you are redirected to the documentation. To go to the service's UI instead, disable the OAuth2 Proxy provider button by setting `OAUTH2_PROXY_SKIP_PROVIDER_BUTTON=true`.
 
-  ```bash
-  kubectl -n kyma-system rollout restart deployment tracing-auth-proxy
-  ```
+3. Restart the OAuth2 Proxy pod:
```
Comment (NHingerl) [inline docs/04-operation-guides/security/sec-06-access-expose-grafana.md:72]:
```suggestion
3. Restart the OAuth2 Proxy Pod:
```

File: docs/01-overview/main-areas/observability/obsv-10-useful-links.md [hunk 7]
```diff
@@ -17,10 +17,10 @@ If you're interested in learning more about the Observability area, check out th
 
 - Troubleshoot Observability-related issues:
   - [Prometheus Istio Server keeps crashing](../../../04-operation-guides/troubleshooting/observability/obsv-01-troubleshoot-prometheus-istio-server-crash-oom.md)
-  - [Jaeger shows fewer traces than expected](../../../04-operation-guides/troubleshooting/observability/obsv-02-troubleshoot-jaeger-shows-few-traces.md)
+  - [Tracing backend shows fewer traces than expected](../../../04-operation-guides/troubleshooting/observability/obsv-02-troubleshoot-trace-backend-shows-few-traces.md)
```
Comment (NHingerl) [inline docs/01-overview/main-areas/observability/obsv-10-useful-links.md:20]: Shall we say "tracing backend" or "trace backend" ? Consistency would be great.
Most other docs say "trace backend", so it's less effort to adjust it here.

```suggestion
  - [Trace backend shows fewer traces than expected](../../../04-operation-guides/troubleshooting/observability/obsv-02-troubleshoot-trace-backend-shows-few-traces.md)
```

File: docs/04-operation-guides/troubleshooting/observability/obsv-02-troubleshoot-trace-backend-shows-few-traces.md [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+title: Trace backend doesn't show the traces you want to see
+---
+
+## Condition
+
+Trace backend shows fewer traces than you would like to see.
+
+## Cause
+
+By [default](https://kyma-project.io/docs/kyma/latest/01-overview/main-areas/telemetry/telemetry-03-traces#istio), only 1% of the requests are sent to trace backend for trace recording.
```
Comment (NHingerl) [inline docs/04-operation-guides/troubleshooting/observability/obsv-02-troubleshoot-trace-backend-shows-few-traces.md:11]:
```suggestion
By [default](https://kyma-project.io/docs/kyma/latest/01-overview/main-areas/telemetry/telemetry-03-traces#istio), only 1% of the requests are sent to the trace backend for trace recording.
```

File: docs/04-operation-guides/troubleshooting/observability/obsv-02-troubleshoot-trace-backend-shows-few-traces.md [hunk 9]
```diff
@@ -0,0 +1,36 @@
...
+
+## Remedy
+
+To see more traces in trace backend, increase the percentage of requests by changing the default settings.
```
Comment (NHingerl) [inline docs/04-operation-guides/troubleshooting/observability/obsv-02-troubleshoot-trace-backend-shows-few-traces.md:15]:
```suggestion
To see more traces in the trace backend, increase the percentage of requests by changing the default settings.
```
Did you just search + replace the word "Jaeger"? :D

File: docs/04-operation-guides/troubleshooting/observability/obsv-02-troubleshoot-trace-backend-shows-few-traces.md [hunk 10]
```diff
@@ -0,0 +1,36 @@
...
+If you just want to see traces for one particular request, you can manually force sampling.
+
+### Change the default setting
+
+To override the default percentage, you deploy a YAML file to an existing kyma installation.
```
Comment (NHingerl) [inline docs/04-operation-guides/troubleshooting/observability/obsv-02-troubleshoot-trace-backend-shows-few-traces.md:20]:
```suggestion
To override the default percentage, you deploy a YAML file to an existing Kyma installation.
```

File: docs/05-technical-reference/00-configuration-parameters/svls-02-environment-variables.md [hunk 11]
```diff
@@ -18,13 +18,9 @@ Every runtime provides its own unique environment configuration which can be rea
 | **SERVICE_NAMESPACE** | | The Namespace where the right Function exists on a cluster. |
 | **KUBELESS_INSTALL_VOLUME** | `/kubeless` | Full path to volume mount with users source code. |
 | **FUNC_RUNTIME** | | The name of the actual runtime. Possible values: `python39`, `nodejs16`, `nodejs16`. |
-| **TRACE_COLLECTOR_ENDPOINT** | `http://tracing-jaeger-collector.kyma-system.svc.cluster.local:4318/v1/traces` | Full address of the Open-Telemetry Trace Collector. |
-| **JAEGER_SERVICE_ENDPOINT** | `http://tracing-jaeger-collector.kyma-system.svc.cluster.local:14268/api/traces` | **Deprecated in 2.8** Full address of the Jaeger service. |
+| **TRACE_COLLECTOR_ENDPOINT** | | Full address of the Open-Telemetry Trace Collector is exported if trace collector endpoint is present |
```
Comment (NHingerl) [inline docs/05-technical-reference/00-configuration-parameters/svls-02-environment-variables.md:21]:
```suggestion
| **TRACE_COLLECTOR_ENDPOINT** | | Full address of the OpenTelemetry Trace Collector is exported if trace collector endpoint is present |
```

File: docs/03-tutorials/00-serverless/svls-11-set-asynchronous-connection-of-functions.md [hunk 12]
```diff
@@ -84,7 +84,7 @@ This tutorial shows only one possible use case. There are many more use cases on
...
 
    >**NOTE:** The event object provides convenience functions to build and publish events. To send the event, build the Cloud Event. To learn more, read [Function's specification](../../05-technical-reference/svls-08-function-specification.md#event-object-sdk). In addition, your **eventOut.source** key must point to `“kyma”` to use Kyma in-cluster Eventing.
-   >**NOTE:** There is a `require('axios')` line even though the Function code is not using it directly. This is needed for the auto-instrumentation to properly handle the outgoing requests sent using the `publishCloudEvent` method (which uses `axios` library under the hood). Without the `axios` import the Function still works, but the published events are not reflected in Jaeger.
+   >**NOTE:** There is a `require('axios')` line even though the Function code is not using it directly. This is needed for the auto-instrumentation to properly handle the outgoing requests sent using the `publishCloudEvent` method (which uses `axios` library under the hood). Without the `axios` import the Function still works, but the published events are not reflected in trace backend.
```
Comment (NHingerl) [inline docs/03-tutorials/00-serverless/svls-11-set-asynchronous-connection-of-functions.md:87]:
```suggestion
   >**NOTE:** There is a `require('axios')` line even though the Function code is not using it directly. This is needed for the auto-instrumentation to properly handle the outgoing requests sent using the `publishCloudEvent` method (which uses `axios` library under the hood). Without the `axios` import the Function still works, but the published events are not reflected in the trace backend.
```

File: docs/03-tutorials/00-serverless/svls-12-customize-function-traces.md [hunk 13]
```diff
@@ -109,12 +110,5 @@ The following code samples illustrate how to enrich the default trace with custo
...
-3. [Expose Jaeger securely](../../04-operation-guides/security/sec-06-access-expose-grafana.md) and open the following Jaeger's address in your browser:
 
-   ```text
-   http://localhost:16686
-   ```
-
-   > **NOTE:** By default, only 1% of the requests are sent to Jaeger for the trace recording. To increase this number see the [Jaeger doesn't show the traces you want to see](../../04-operation-guides/troubleshooting/observability/obsv-02-troubleshoot-jaeger-shows-few-traces.md) page.
-
-4. Choose your Deployment's name from the **Service** list and click **Find Traces**
+3. Find the traces for the function in the trace backend.
```
Comment (NHingerl) [inline docs/03-tutorials/00-serverless/svls-12-customize-function-traces.md:114]:
```suggestion
3. Find the traces for the Function in the trace backend.
```

File: docs/05-technical-reference/00-architecture/obsv-03-architecture-tracing.md [hunk 14]
```diff
@@ -2,28 +2,16 @@
 title: Tracing Architecture
 ---
 
-Kyma uses a Jaeger-based tracing component to collect and query traces.
-
-> **NOTE:** Jaeger is [deprecated](https://kyma-project.io/blog/jaeger-deprecation) and is planned to be removed. If you want to install a custom Jaeger stack, take a look at [Install custom Jaeger in Kyma](https://github.com/kyma-project/examples/tree/main/jaeger).
-
-Collection and query may occur at the same time. This way, you can inspect specific traces using the Jaeger UI, while Jaeger takes care of proper trace collection and storage in parallel.
+With the stratergy shift to more [flexibility](https://blogs.sap.com/2022/09/25/from-observability-to-telemetry-a-strategy-shift-in-sap-btp-kyma-runtime/), a layer or collector is introduced for collecting trace data. This collector pushes the trace data to [configured](../../01-overview/main-areas/telemetry/telemetry-03-traces.md#setting-up-a-tracepipeline) trace backend. The user can bring his own trace backend or [Install custom Jaeger in Kyma](https://github.com/kyma-project/examples/tree/main/jaeger). One can inspect specific traces using the trace backend.
```
Comment (NHingerl) [inline docs/05-technical-reference/00-architecture/obsv-03-architecture-tracing.md:5]:
```suggestion
With the strategy shift to more [flexibility](https://blogs.sap.com/2022/09/25/from-observability-to-telemetry-a-strategy-shift-in-sap-btp-kyma-runtime/), a layer or collector is introduced for collecting trace data. This collector pushes the trace data to the [configured trace backend](../../01-overview/main-areas/telemetry/telemetry-03-traces.md#setting-up-a-tracepipeline). You can bring your own trace backend or [Install custom Jaeger in Kyma](https://github.com/kyma-project/examples/tree/main/jaeger). You can inspect specific traces using the trace backend.
```

File: docs/05-technical-reference/00-architecture/obsv-03-architecture-tracing.md [hunk 15]
```diff
@@ -2,28 +2,16 @@
...
+The process of collecting traces by custom Jaeger deployed in Kyma cluster looks as follows:
 
 1. The application receives a request, either from an internal or external source.
-2. If the application has Istio injection enabled and HTTP headers are missing, [Istio proxy](https://github.com/istio/proxy) enriches the request with the correct HTTP headers and propagates them to the Application container. Furthermore, Istio proxy sends the trace data for any intercepted request the otel-collector instance served by the [Telemetry module](./../../01-overview/main-areas/telemetry/README.md), which is configured by default to ship the trace data to Jaeger using the OTLP protocol.  
+2. If the application has Istio injection enabled and HTTP headers are missing, [Istio proxy](https://github.com/istio/proxy) enriches the request with the correct HTTP headers and propagates them to the Application container. Furthermore, Istio proxy sends the trace data for any intercepted request the otel-collector instance served by the [Telemetry module](./../../01-overview/main-areas/telemetry/README.md), which is configured by default to ship the trace data to trace backend using the OTLP protocol.
```
Comment (NHingerl) [inline docs/05-technical-reference/00-architecture/obsv-03-architecture-tracing.md:16]:
```suggestion
2. If the application has Istio injection enabled and HTTP headers are missing, [Istio proxy](https://github.com/istio/proxy) enriches the request with the correct HTTP headers and propagates them to the Application container. Furthermore, Istio proxy sends the trace data for any intercepted request the otel-collector instance served by the [Telemetry module](./../../01-overview/main-areas/telemetry/README.md), which is configured by default to ship the trace data to the trace backend using the OTLP protocol.  
```

**Review by NHingerl — APPROVED** [review]

**Pull request merged**